.github/workflows/merlin.yml

```diff
@@ -283,17 +282,25 @@ jobs:
           path: merlin.${{ needs.create-version.outputs.version }}.tar
           retention-days: ${{ env.ARTIFACT_RETENTION_DAYS }}
 
-  build-batch-predictor-base:
+  build-batch-predictor-base-image:
     runs-on: ubuntu-latest
     needs:
       - create-version
       - test-batch-predictor
     env:
       DOCKER_IMAGE_TAG: "ghcr.io/${{ github.repository }}/merlin-pyspark-base:${{ needs.create-version.outputs.version }}"
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - name: Build Batch Predictor Base Docker
         run: docker build -t ${{ env.DOCKER_IMAGE_TAG }} -f python/batch-predictor/docker/base.Dockerfile python
+      - name: Test Build Batch Predictor Docker
```

Comment:
I'm adding a new step here instead of a separate job to test the image-building process of the batch predictor model. Having it as a separate job require us to unpack the docker build package first and would be a little bit slower -- since we need to spawn a different job.